### PR TITLE
Symfony Setup Checklist

### PR DESCRIPTION
**Pull request by emr**

### CODE REVIEW

File: checklists/symfony-setup-checklist.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Setting Up a New Symfony Project from Scratch
+category: Back-End
+date: "2020-03-11"
+tags: ['php', 'symfony']
+description: Steps we follow when starting a new project with Symfony. We recommend you to commit your changes in VCS you using after each step. Every to-do title can be a commit message for you.
```
Comment (ecempehlivan) [inline checklists/symfony-setup-checklist.md]:
```suggestion
description: Steps we follow when starting a new project with Symfony. We recommend you to commit your changes in VCS you use after each step. Every To-Do title can be a commit message.
```

File: checklists/symfony-setup-checklist.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+- [ ] Create the initial Symfony project  
+Install symfony via `composer create-project` using `symfony/skeleton` or `symfony/website-skeleton`
```
Comment (ecempehlivan) [inline checklists/symfony-setup-checklist.md]:
```suggestion
Install symfony via `composer create-project` by using `symfony/skeleton` or `symfony/website-skeleton`
```

File: checklists/symfony-setup-checklist.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- [ ] Set the coding standards  
+Add php-cs-fixer via composer and add cs rules to `.php_cs.dist` file
+
+- [ ] Configure phpstan  
+Add phpstan static analyzer tool via composer and configure it by adding the extensions for Symfony
+
+- [ ] Create the docker configuration  
+Make sure to project running when docker container is up
```
Comment (ecempehlivan) [inline checklists/symfony-setup-checklist.md]:
```suggestion
Make sure that the project is running when docker container is up
```

File: checklists/symfony-setup-checklist.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- [ ] Create the Makefile  
+Create a Makefile which includes the steps applied frequently in the project
```
Comment (ecempehlivan) [inline checklists/symfony-setup-checklist.md]:
```suggestion
Create a Makefile which includes the steps that applied frequently in the project
```

File: checklists/symfony-setup-checklist.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+description: Steps we follow when starting a new project with Symfony. We recommend you to commit your changes in VCS you using after each step. Every to-do title can be a commit message for you.
+---
+
+- [ ] Create the initial Symfony project  
+Install symfony via `composer create-project` using `symfony/skeleton` or `symfony/website-skeleton`
+
+- [ ] Set the coding standards  
+Add php-cs-fixer via composer and add cs rules to `.php_cs.dist` file
+
+- [ ] Configure phpstan
```
Comment (akemalakcay) [inline checklists/symfony-setup-checklist.md]:
```suggestion
- [ ] Configure PHPStan  
```

File: checklists/symfony-setup-checklist.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] Create the initial Symfony project  
+Install symfony via `composer create-project` using `symfony/skeleton` or `symfony/website-skeleton`
+
+- [ ] Set the coding standards  
+Add php-cs-fixer via composer and add cs rules to `.php_cs.dist` file
+
+- [ ] Configure phpstan  
+Add phpstan static analyzer tool via composer and configure it by adding the extensions for Symfony
+
+- [ ] Create the docker configuration
```
Comment (akemalakcay) [inline checklists/symfony-setup-checklist.md]:
```suggestion
- [ ] Create the Docker configuration  
```

File: checklists/symfony-setup-checklist.md (hunk 3 above)
Comment (akemalakcay) [inline checklists/symfony-setup-checklist.md]:
```suggestion
Make sure to project is running when Docker container is up
```

File: checklists/symfony-setup-checklist.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+Make sure to project running when docker container is up
+
+- [ ] Create the Makefile  
+Create a Makefile which includes the steps applied frequently in the project
+
+- [ ] Configure the CI workflow  
+Create a project on CircleCI and configure the CI workflow
+
+- [ ] Create the README.md file  
+Create a README.md file which includes information about how to install and up the project, the project's code style, the jira project link and contributing rules and steps. **And don't forget to hold this file up to date** 🤓
```
Comment (alpcanaydin) [inline checklists/symfony-setup-checklist.md]:
```suggestion
Create a README.md file which includes information about how to install and up the project, the project's code style, the Jira project link and contributing rules and steps. **And don't forget to hold this file up to date** 🤓
```

File: checklists/symfony-setup-checklist.md (hunk 4 above)
Comment (ahmetcelikezer) [inline checklists/symfony-setup-checklist.md]:
```suggestion
Create a Makefile which includes the steps which should be applied while deploying the project and frequently running in the project
```
We can consider including the steps we write for the steps that must be applied deployment.

Comment (emr) [reply checklists/symfony-setup-checklist.md]: We collect the commands which frequently used for a task such as building assets, linting source code, preparing the application, etc in Makefile. So not only deployment task.

**Pull request merged**